SPECS/graphviz/graphviz.spec

```diff
@@ -265,7 +267,7 @@ sed -i 's|_MY_JAVA_INCLUDES_|-I%{java_home}/include/ -I%{java_home}/include/linu
 
 %configure --with-x --disable-static --disable-dependency-tracking \
 	--without-mylibgd --with-ipsepcola --with-pangocairo \
-	--without-gdk-pixbuf --with-visio --disable-silent-rules \
+	--without-gdk-pixbuf --with-visio --disable-silent-rules --with-freetypeincludedir="%{_includedir}/freetype2" --with-freetypelibdir="%{_libdir}" \
```

Comment:
Double-checking: should it be `freetype` of `freefont`? It see that these are names for two separate packages, so I got curious.

Reply:
I did mean `freetype`, but now I'm investigating the mess that is graphviz font configuration since it seems like it might have changed a little.